assets/src/edit-story/icons/union.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="29" height="19" viewBox="0 0 29 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M22.9214 -0.000865933L28.0008 -0.000594195C28.2769 -0.000579413 28.5008 0.22329 28.5008 0.499433C28.5008 0.775575 28.2769 0.999421 28.0007 0.999406L22.9213 0.999134C22.4834 0.999111 22.0673 1.19048 21.7823 1.52302L16.3808 7.82561L16.3807 7.82564L15.3819 8.99083L19.5396 13.6679L22.0543 16.4966C22.3389 16.8168 22.7469 17 23.1753 17H28.5C28.7761 17 29 17.2239 29 17.5C29 17.7761 28.7761 18 28.5 18H23.1753C22.4613 18 21.7813 17.6947 21.3069 17.161L18.7922 14.3323L14.7251 9.75708L10.3803 14.8255L7.97875 17.6271C7.5038 18.1812 6.81046 18.5 6.08068 18.5H1C0.723858 18.5 0.5 18.2762 0.5 18C0.5 17.7239 0.723858 17.5 1 17.5H6.08068C6.51855 17.5 6.93455 17.3087 7.21952 16.9763L9.62109 14.1747L14.0545 9.00279L12.5757 7.33927L6.95108 1.46286C6.66811 1.16721 6.27663 1.00003 5.86738 1.00005L0.500026 1.00032C0.223883 1.00034 1.42462e-05 0.776493 6.79898e-10 0.50035C-1.42448e-05 0.224208 0.223832 0.000338886 0.499974 0.00032464L5.86733 4.77767e-05C6.5494 1.25802e-05 7.20187 0.27866 7.6735 0.771395L13.3045 6.65446C13.3087 6.65889 13.3129 6.6634 13.317 6.66799L14.7114 8.23654L15.6215 7.17486L15.6215 7.17482L21.023 0.87227C21.498 0.318051 22.1915 -0.000904974 22.9214 -0.000865933Z" fill="white" fill-opacity="0.24"/>
```

Comment:
I think we should change the `fill` to `currentColor` and remove the `fill-opacity` here - to make the outside context define these properties instead.